beluga_amcl/docs/PARAMETERS.md

```diff
@@ -0,0 +1,75 @@
+# Beluga AMCL Parameter Reference
+
+## ROS Interface Parameters
+
+| Parameter | Description | Navigation 2 AMCL | Beluga AMCL |
+|-----------|-------------|:-----------------:|:-----------:|
+| `base_frame_id` | Robot base frame name rigidly attached to the mobile robot base. | ✅ | ✅ |
+| `odom_frame_id` | Odometry frame name. The pose of a mobile platform relative to this frame can drift over time but it must be continuous (without discrete jumps). | ✅ | ✅ |
+| `global_frame_id` | Map frame name. The pose of a mobile platform relative to this frame does not drift significantly over time but it can change in discrete jumps at any time. This node can estimate and publish the transform between global and odometry frames. | ✅ | ✅ |
+| `scan_topic` | The name of the topic where laser scans are being published. | ✅ | ✅ |
+| `map_topic` | The name of the topic where the map is published by the map server. | ✅ | ✅ |
+| `initial_pose_topic` | The name of the topic where an initial pose can be published. _Navigation 2 supports a fixed subscriber in the `initialpose` topic._ | | ✅ |
```

Comment:
@nahueespinosa hmm, what do you mean here? Topics can be [remapped](https://design.ros2.org/articles/static_remapping.html).

Reply:
Reworded here https://github.com/Ekumen-OS/beluga/pull/196/commits/b0672a7827aa45034ea11d6fa2404de3df918006. There is no parameter in Nav2 AMCL.